src/main/java/com/oltpbenchmark/api/BenchmarkModule.java

```diff
@@ -91,6 +102,22 @@ public final Connection makeConnection() throws SQLException {
         }
     }
 
+    private HikariDataSource hikariDataSource;
+
+    public void createDataSource() {
+        HikariConfig config = new HikariConfig();
```

Comment:
We should shutdown the dataSource and its associated pool after the test is done. You can call datasource.close for the same. 


<br class="Apple-interchange-newline">

Reply:
implemented closeDataSource() method which will close it in ThreadBench finalizeWorkers()